views/index.erb

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script src="https://cdn.tailwindcss.com"></script>
+  <title>Document</title>
+</head>
+<body>
+<!--login nav bar-->
+  <nav class="bg-gradient-to-r from-blue-300 to-blue-600 px-2 sm:px-4 py-2.5 rounded text-white">
+  <% if logged_in? %>
+    <div class="flex items-center justify-between">
+      <a href="/"><img class = "h-10 w-10" src = "https://cdn-icons-png.flaticon.com/512/3338/3338265.png" alt =""/></a>
+      <!--Menu Items-->
+      <div class="flex space-x-6">
+        <a href="/">Home</a>
+        <a href="/users/<%= session[:user_id] %>">Profile</a>
+        <a href="/sessions/logout">Log Out</a>
+      </div>
+    </div>
+  <% else %>
+    <div class="flex items-center justify-between">
+      <a href="/"><img class = "h-10 w-10" src = "https://cdn-icons-png.flaticon.com/512/3338/3338265.png" alt ="" /></a>
+      <!--Menu Items-->
+      <div class="flex space-x-6 pr-4">
+        <a href="/">Home</a>
+        <a href="/users/new">Sign Up</a>
+        <a href="/sessions/new">Log In</a>
+      </div>
+    </div>
+  <% end %>
+  </nav>
+
+  <% if @alert != nil %>
+  <div class="flex items-center bg-blue-500 text-white text-sm font-bold px-4 py-3 rounded" role="alert">
+    <svg class="fill-current w-4 h-4 mr-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M12.432 0c1.34 0 2.01.912 2.01 1.957 0 1.305-1.164 2.512-2.679 2.512-1.269 0-2.009-.75-1.974-1.99C9.789 1.436 10.67 0 12.432 0zM8.309 20c-1.058 0-1.833-.652-1.093-3.524l1.214-5.092c.211-.814.246-1.141 0-1.141-.317 0-1.689.562-2.502 1.117l-.528-.88c2.572-2.186 5.531-3.467 6.801-3.467 1.057 0 1.233 1.273.705 3.23l-1.391 5.352c-.246.945-.141 1.271.106 1.271.317 0 1.357-.392 2.379-1.207l.6.814C12.098 19.02 9.365 20 8.309 20z"/></svg>
```

Comment:
The long class names are a bit hard to read, but I guess they describe exactly what the CSS is doing